recipe/install_ucx.sh

```diff
@@ -19,7 +24,8 @@ export LDFLAGS="${LDFLAGS} -L${CONDA_BUILD_SYSROOT}/usr/lib64"
     --disable-cma \
     --enable-mt \
     --with-gnu-ld \
-    --with-rdmacm="/usr"
+    --with-rdmacm="/usr" \
```

Comment:
Is the assumption that rdmacm is always available in `/usr/` justified? (unless we actually place it there)

Reply:
We are building in a Docker container where it is placed there. Have you seen it in other locations? If so, under what circumstances does it show up in another location?

Reply:
I've seen `librdmacm.so` show up in `/lib64` for instance. I don't know what other exceptions exist though.

Reply:
Well this is really only specifying the location for the build.

At runtime the standard library loader behavior prevails, which AIUI would check locations like `/lib64`.

ref: https://linux.die.net/man/8/ld-linux

Reply:
You need to use a CDT for this.

Reply:
There is [one]( https://github.com/conda-forge/ucx-split-feedstock/blob/39462ed89778358f5e60ed7611f97dc2a4bdea2c/recipe/meta.yaml#L44 ). Anyways this is getting a bit off topic. [This line]( https://github.com/conda-forge/staged-recipes/pull/9162/files#diff-1c214c0d6ef4844a1d546d3a0126a9b9R22 ) was in the original `staged-recipes` submission ( https://github.com/conda-forge/staged-recipes/pull/9162 ). If we want to discuss it further (which is totally fine by me), would suggest raising a new issue.

Reply:
Have submitted PR ( https://github.com/conda-forge/ucx-split-feedstock/pull/15 ), which drops this path. Looks like that works fine. Though please take a look. 🙂